overrides/resources/crafttweaker/lang/en_us.lang

```diff
@@ -528,7 +528,7 @@ dj2.atum_artifact_melee.desc0=§4§lConsidered as an Atum Melee Artifact.
 dj2.atum_artifact_ranged.desc1=§4§lConsidered as an Atum Ranged Artifact.
 dj2.atum_artifact_armor.desc2=§4§lConsidered as an Atum Armor Artifact.
 dj2.atum_artifact_necklace.desc3=§4§lConsidered as an Atum Necklace Artifact.
-dj2.relic_ore.desc0=§fMine it up regularly for §r§4§lDirty Idols, §r§for use
+dj2.relic_ore.desc0=§fMine it up to regularly obtain §r§4§lDirty Idols, §r§for use
```

Comment:
"regularly" meant "without Silk Touch, just a normal pick". this edit appears to presume that "regularly" was a typo, and meant "Dirty Idols were obtained frequently", and the sentence no longer makes sense.
should either be reverted, or changed to `Mine it up regularly to obtain`

Reply:
That is indeed how I understood that sentence. I could change the "regularly" to "normally", which would be less confusing imo since it is then more in contrast with the Silk Touch mentioned in the next line.